pkg/broker/broker.go

```diff
@@ -101,16 +101,15 @@ func (a AnsibleBroker) getServiceInstance(instanceUUID uuid.UUID) (*apb.ServiceI
 
 }
 func (a AnsibleBroker) Login() error {
-	a.log.Debug("Retrieving serviceaccount token")
-	token, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
+	clientConfig, err := k8srestclient.InClusterConfig()
 	if err != nil {
-		a.log.Debug("Error reading serviceaccount token")
+		a.log.Warning("Failed to create a restclient.InClusterConfig: %v.", err)
 		return err
 	}
 
-	return apb.OcLogin(a.log, "https://kubernetes.default",
-		"--certificate-authority", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
-		"--token", string(token),
+	return apb.OcLogin(a.log, clientConfig.Host,
+		"--certificate-authority", clientConfig.CAFile,
```

Comment:
I'm -1 to merge this part.  OcLogin is only used by a local broker so we can't use an incluster config object.  In addition, I think we should remove RefreshLoginToken, which is the function that calls this. IMO we shouldn't change anything here, but follow up with a patch to remove RefreshLoginToken and OcLogin for the TP. I'll come up with a patch for this shortly.